spec/mongo/collection/view_spec.rb

```diff
@@ -545,11 +688,9 @@
             end
 
             it 'overrides the modifier value with the option value' do
-              expect(query_spec[:selector][:$query]).to eq(selector)
+              expect(query_spec[:selector][:$query]).to eq(options[:modifiers][:$query])
```

Comment:
@behackett If you have a $query in modifiers, does that get used instead of the selector?

for example:
coll.find({ a: 1 }, { :modifiers => { :$query => { b: 2 } } )